### ISSUE TITLE
FIrefox restore session tab name

### ISSUE THREAD

**Issue by axelKeizoStahl** [josh-berry/tab-stash#539]
### OS Version

Arch Linux

### Browser Version

Firefox 130.0.1

### Tab Stash Version

3.1

### Installed Extensions

- ublock origin
- vimium
- BlockChan
- Dark Reader
- 4chan faster images

### Problem Description

When firefox has its restore session prompt, and you click restore, the name of the current tab is changed to session restore for a moment. But then once that tab loads it changes back to the actual title of the tab. 
However, tab stash still labels it as session restore.

### Steps to Reproduce

1. quit out of firefox 
2. get the session restore prompt and accept
3. tab stash labels the current tab as "Session Restore"

### Expected Behavior

Instead, it should change to the actual title of that site once the site loads.

### Vote for This Issue

- [X] _Readers: If you are also experiencing this problem, please vote for it by giving the ORIGINAL POST a thumbs-up using the :smiley: button below.  You are welcome to leave comments and discuss the issue if you have more details to add, but "Me too!" comments are not counted by the voting system._

**Comment by josh-berry** [comment]
Afraid I'm not able to reproduce this. How are you getting the restore session prompt to appear?

**Comment by axelKeizoStahl** [comment]
Revised steps to get restore session:
- run firefox and open some tabs
- run `killall firefox` to stop firefox
- open firefox back up, and you should be greeted with a restore session prompt

**Comment by axelKeizoStahl** [comment]
![image](https://github.com/user-attachments/assets/2d647d50-6f09-4b0f-9c9f-35252d02522d)
Here is an example, as you can see the focused tab is youtube, and yet it is still displaying Restore Session

**Comment by josh-berry** [comment]
Got it, thanks. Killing Firefox doesn't bring up the "Restore Session" prompt for me, it just restores the session and so I don't see this.

Without a way to reproduce this, I have no way to confirm it, but it's quite likely the issue is that Firefox is just not reporting the change in tab title when you click "Restore". If that's actually the case, there's unfortunately not much I can do about it.

Does reloading the Tab Stash UI fix the tab title? You can reload it by closing and reopening the sidebar, or by switching to a different sidebar and then switching back to Tab Stash again.

**Comment by axelKeizoStahl** [comment]
Yeah, reloading the Tab Stash UI is what I've been doing so far, its too bad that there isn't really a good fix. Thanks for the help :+1:
